mmv1/templates/terraform/custom_delete/active_directory_domain.erb

```diff
@@ -0,0 +1,52 @@
+if d.Get("deletion_protection").(bool) {
```

Comment:
This logic should be able to go in a pre_delete by itself to prevent having to copy over the entire delete functionality

Reply:
Updated. Thanks!